### PR TITLE
[Html to Design] -> [UI Graphics]

### PR DESCRIPTION
**Pull request by devIbtihadj**
https://www.figma.com/community/plugin/1159123024924461424/html.to.design

### CODE REVIEW

**Review by Alaev — CHANGES_REQUESTED** [review]
Please remove .idea folder and if you can add the resource without linting the file.

**Comment by devIbtihadj** [comment]
done

File: readme.md [hunk 1]
```diff
@@ -1341,7 +1341,8 @@
 | [devices.css](http://marvelapp.github.io/devices.css/) | Pure CSS phones and tablets by Marvel App. |
 | [Troopl](https://troopl.com) | Build and publish a free portfolio in minutes. |
 | [Apifox](https://apifox.cn) | Apifox = Postman + Swagger + Mock + JMeter |
-| [A Modern CSS Reset](https://piccalil.li/blog/a-modern-css-reset/) | Resets the css styling of all HTML elements to a consistent baseline across browsers.
+| [A Modern CSS Reset](https://piccalil.li/blog/a-modern-css-reset/) | Resets the css styling of all HTML elements to a consistent baseline across browsers. |
+| [Github profile readme generator](https://rahuldkjain.github.io/gh-profile-readme-generator/) | Allow you to generate a github profile reamdme. |
```
Comment (Alaev) [inline readme.md:1345]: Please add resource one in a separate PR.

**Review by Alaev — CHANGES_REQUESTED** [review]
Please see comment.

**Comment by devIbtihadj** [comment]
Ok, i think it is ok now, you can check it

**Review by Alaev — APPROVED** [review]

**Pull request merged**